src/rgw/rgw_process.cc

```diff
@@ -305,6 +306,8 @@ int process_request(rgw::sal::RGWRadosStore* const store,
     *http_ret = s->err.http_ret;
   }
   int op_ret = 0;
+  *user = s->user->get_id().to_str();
+
```

Comment:
if you're taking the string as a pointer, you need to check for null before assigning to it. `RGWLoadGenProcess::handle_request()` is passing null here (and, if civetweb isn't using the value, it should too)